index.js

```diff
@@ -1,18 +1,133 @@
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
+//share files within project
+const lib = require('./ask');
+
+//choosing which game to play first
+gameChoice();
+async function gameChoice() {
+  let userGameChoice = await lib.ask('Do you want to guess first or should I? \nPress "1" if you want to guess first and press "2" if you want me to guess first. \n')
+  if (parseInt(userGameChoice) === 1) {
+    startHumGuessNum();
+  } else if (parseInt(userGameChoice) === 2) {
+    startCompGuessNum();
+  } else {
+    console.log("Well that's not a choice at all! I guess you don't want to play any of my games :( Goodbye!");
+    process.exit();
+  };
 
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
 
-start();
+};
 
-async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
+//computer guesses number
+async function startCompGuessNum() {
+  //range
+  let high = (process.argv[2] || 100);
```

Comment:
Good use of process.argv to set the range.  Could you possibly integrate this as a question inside the game instead though?  That way a user that doesn't know the code would be able to set the range?